### DISCUSSION TITLE
chromosomal diseases

### DISCUSSION THREAD

**Discussion by nicolevasilevsky** [monarch-initiative/mondo#2584] [Mondo design decisions]
Action items (per @cmungall)
 - release monochrom (this is useful for many things including axiomatizing chromosomal disorders)
 - a trivial owl class representation of major structural variants such as those found in civicdb including classic ones like philadelphia chromosome.

**Comment by nicolevasilevsky** [comment]
From @pnrobinson 
There are very difficult modeling issues that involve the boundaries of chromosomal disorders – essentially, while there are some recurrent diseases that are said to be always the same, this is not really the case. Even with the CML example, the precise breakpoint can differ. For many Mendelian diseases, the extent of the deletion (or whatever) is related to the severity of the diseases and the phenotypic manifestations. For relatively common diseases such as NF1 deletion or Williams Beuren, though, they are generally called the same disease. For single case reports or rarer disorders, it will probably require our biocuration to identify commonalities etc.

**Comment by nicolevasilevsky** [comment]
related ticket: https://github.com/monarch-initiative/mondo/issues/7

**Comment by sabrinatoro** [comment]
The classification of the chromosomal anomaly terms is not consistent. 
Proposal in this issue #3160
